lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntime.cpp

```diff
@@ -2937,4 +2940,87 @@ llvm::Expected<lldb::addr_t> GetTaskAddrFromThreadLocalStorage(Thread &thread) {
   return task_addr;
 #endif
 }
+
+namespace {
+
+struct TaskStatusRecord {
```

Comment:
Maybe add a Doxygen comment to explain what this is used for?

Reply:
done